install/resources/upgrade43.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * MyBB 1.8
+ * Copyright 2014 MyBB Group, All Rights Reserved
```

Comment:
Nitpick: Copyright header is 2014 and for 1.8

Reply:
Ample of files are that way in MyBB, I guess it represents what generally declared as `@since` with version number